src/pages/permissions/actions/GroupActions.tsx

```diff
@@ -29,7 +29,12 @@ const GroupActions: FC<Props> = ({ group }) => {
               panelParams.openEditGroup(group.name);
             }}
             type="button"
-            title="Edit group"
+            title={
+              canEditGroup(group)
+                ? "Edit group"
+                : "Edit group - You do not have permission to edit this group"
+            }
+            disabled={!canDeleteGroup(group)}
```

Comment:
```suggestion
            disabled={!canEditGroup(group)}
```
I think this is a mixup.